docs/src/tutorials/hybrid-applications-with-webpack/using-your-library.md

```diff
@@ -64,6 +64,14 @@ import("../crate/pkg").then(module => {
 });
 ```
 
+## Install `npm` Dependencies
+
+Before your first run, don't forget to install npm dependencies in your project's root directories:
```

Comment:
```suggestion
Before your first run, don't forget to install npm dependencies in your project's root directory:
```
Shouldn't this be "project's root directory" (no plural)? Because in line 77 it is also in singular form:
https://github.com/rustwasm/wasm-pack/pull/892/files#diff-4fab3c527d81230563c65b7a11c52ac2R77